bazaarvoice/emodb
# UPGRADE DROPWIZARD!!!!!

## Github Issue #

`None`

## What Are We Doing Here?

We are upgrading nearly every dependency that we got. Ladies and gentlemen, Emo is moving into the 21st century.

## How to Test and Verify

1. Check out this PR
2. Run Locally
3. Make Sure nothing changed whatsoever

## Risk

### Level 

`High`

### Required Testing

`The Whole Kitchen Sink`

### Risk Summary

This is ridiculously high risk. We are upgrading nearly every dependency in Emo, and the number of possible things that could go wrong or endless. 

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
